doc/lua_api.txt

```diff
@@ -3326,18 +3330,23 @@ This is basically a reference to a C++ `ServerActiveObject`
       }
 
 ##### LuaEntitySAO-only (no-op for other objects)
-* `set_velocity({x=num, y=num, z=num})`
-* `get_velocity()`: returns `{x=num, y=num, z=num}`
-* `set_acceleration({x=num, y=num, z=num})`
-* `get_acceleration()`: returns `{x=num, y=num, z=num}`
+* `set_velocity(vel)`
+    * `vel` is a vector, e.g. `{x=0, y=2.3, z=1}`
+* `get_velocity()`: returns the velocity, a vector
+* `set_acceleration(acc)`
+    * `acc` is a vector
+* `get_acceleration()`: returns the acceleration, a vector
 * `set_yaw(radians)`
 * `get_yaw()`: returns number in radians
 * `set_texture_mod(mod)`
 * `get_texture_mod()` returns current texture modifier
-* `set_sprite(p={x=0,y=0}, num_frames=1, framelength=0.2,
-  select_horiz_by_yawpitch=false)`
+* `set_sprite(p, num_frames, framelength, select_horiz_by_yawpitch)`
     * Select sprite from spritesheet with optional animation and DM-style
       texture selection based on yaw relative to camera
+    * `p`: table {x=int,y=int}, default: `{x=0, y=0}`
```

Comment:
'p' needs explaining

Reply:
What is DM?

Reply:
@paramat Dungeon Master, from the C++ mobs years ago.

Reply:
idk what p and DM mean, l don't use this function. Maybe DM means Dungeon Master…

Reply:
LOL.

Reply:
'p' is probably the co-ordinate of the frame in the spritesheet.